src/dune_lang/template.mli

```diff
@@ -29,3 +29,26 @@ val pp_split_strings : Format.formatter -> t -> unit
 val remove_locs : t -> t
 
 val to_dyn : t -> Dyn.t
+
+module Var : sig
```

Comment:
If you move this signature to the top, you can get rid of the `type t = var` alias.

Reply:
Well spotted. I moved it and also moved a few pform related functions to it.